main.go

```diff
@@ -24,10 +24,12 @@ var (
 func init() {
 	// command-line flags
 	flagDisableLogin := flag.Bool("disable-login", false, "Disable login page. Turn off authentication.")
+        flagBindAddress := flag.String("bind-address", "0.0.0.0:5000", "Address:Port to which the app will be bound.")
```

Comment:
Can you adjust the code indentation?

Reply:
Uh weird. I used tabs though. I am on mobile right now, I will fix that later today. ;)